### PR TITLE
add fetch running executions of a flow api call

### PR DESCRIPTION
**Pull request by rrrahal**
Add the API Call, according to the [docs](https://azkaban.readthedocs.io/en/latest/ajaxApi.html#fetch-running-executions-of-a-flow)

If anything seems wrong, please let me know (:1st_place_medal:

### CODE REVIEW

**Comment by aguimaraesduarte** [comment]
Thanks for the PR, @rrrahal ! Can you update the README with the new command as well? Looks good otherwise

**Review by aguimaraesduarte — CHANGES_REQUESTED** [review]
Thanks for the PR, @rrrahal ! Can you update the README with the new command as well? Looks good otherwise

**Comment by dmvieira** [comment]
Yes, please just add here: https://github.com/globocom/azkaban-cli/blob/master/README.md new command @rrrahal ... Thank you for your work!

**Comment by rrrahal** [comment]
Done guys, sorry it took me so long. Exams on the university ):

**Review by aguimaraesduarte — APPROVED** [review]

File: azkaban_cli/api.py [hunk 1]
```diff
@@ -560,4 +560,28 @@ def fetch_flow_execution_updates_request(session, host, session_id, exec_id, las
 
     logging.debug("Response: \n%s", response.text)
 
+    return response
+
+def fetch_running_executions_of_a_flow_request(session_id, project, flow):
```
Comment (aguimaraesduarte) [inline azkaban_cli/api.py]: You need to pass in the `session` and `host` as well!

File: azkaban_cli/azkaban.py [hunk 2]
```diff
@@ -715,3 +716,30 @@ def fetch_executions_of_a_flow(self, project, flow, start, length):
...
+        :type project: str
+        :param flow: Flow id on Azkaban
+        :type flow: str
+        :raises FetchRunningExecutionsOfAFlowError: when Azkaban api returns error in response
+        """
+
+        self.__check_if_logged()
+
+        response = api.fetch_running_executions_of_a_flow_request(
+            self.__session_id,
```
Comment (aguimaraesduarte) [inline azkaban_cli/azkaban.py:829]: Pass in `self.__session` and `self.__host` as well

**Review by aguimaraesduarte — CHANGES_REQUESTED** [review]

File: azkaban_cli/azkaban.py (hunk 2 above)
Comment (rrrahal) [reply azkaban_cli/azkaban.py:829]: Done!

File: azkaban_cli/azkaban_cli.py [hunk 3]
```diff
@@ -614,6 +628,15 @@ def fetch_flow_execution_updates(ctx, execution_id, last_update_time):
     """Fetch flow execution updates"""
     __fetch_flow_execution_updates(ctx, execution_id, last_update_time)
 
+@click.command(u'fetch_running_executions_of_a_flow')
+@click.pass_context
+@click.argument(u'project', type=click.STRING)
+@click.argument(u'flow', type=click.STRING)
+def fetch_running_executions_of_a_flow(ctx, project, flow):
+    """ Fetch running executions of a flow"""
+    _fetch_running_executions_of_a_flow(ctz, project, flow)
```
Comment (aguimaraesduarte) [inline azkaban_cli/azkaban_cli.py]: `ctx`!

**Review by aguimaraesduarte — CHANGES_REQUESTED** [review]
Just one quick typo fix, and you also need to add the test cases! Look into the `tests` folder for examples of how to do them. You're very close now

**Comment by aguimaraesduarte** [comment]
@rrrahal Will you be able to work on finishing the PR today? It's the last day of Hacktoberfest!

**Comment by rrrahal** [comment]
I had some problems with conflicts, i will try to fix them later, but i was able to do the tests.

**Review by aguimaraesduarte — APPROVED** [review]

**Comment by aguimaraesduarte** [comment]
@rrrahal I fixed the merge issues for you. Thanks for the contribution!! I'll be merging this now

**Pull request merged**